README.md

```diff
@@ -1 +1,70 @@
-# Math-Magicians
\ No newline at end of file
+# Getting Started with Create React App
+
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+
+## Available Scripts
+
+In the project directory, you can run:
+
```

Comment:
- [ ] Please use this [template ](https://github.com/microverseinc/readme-template/blob/master/README.md)to make your Readme file descriptive.

Reply:
Sorry, I thought we should leave the documentation for the readme file as it is, but I just changed it, thank you very much.